modules/examples/todolist/client/src/main/scala/handlers/PingPongClientHandler.scala

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2017-2018 47 Degrees, LLC. <http://www.47deg.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package examples.todolist.client
+package handlers
+
+import cats.{~>, Monad}
+import cats.Monad.ops._
+import examples.todolist.client.PingPongClient
+import examples.todolist.protocol.Protocols.PingPongService
+import freestyle.rpc.protocol.Empty
+import monix.eval.Task
+import org.log4s._
+
+class PingPongClientHandler[F[_]](
+    implicit M: Monad[F],
+    client: PingPongService.Client[F],
+    T2F: Task ~> F)
```

Comment:
We don't need this natural transformation in this example, we can remove it from all our handlers

Reply:
Totally agree